recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set md5 = "c486211efa29b95771c74f5d8701ca4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy >=1.12
```

Comment:
It appears to not be linking to numpy. I had issues with the conda env being created in circle. It would try to put a version of numpy (1.9.X) which was too old for one of the pythons (3.6 I think, I forgot.) Thus I added this requirement here.